README.md

```diff
@@ -5,6 +5,14 @@
 [![Tests Badge](https://github.com/SAP/cloud-sdk-cli/workflows/Tests/badge.svg?branch=main)](https://github.com/SAP/cloud-sdk-cli/actions)
 [![NPM Downloads](https://img.shields.io/npm/dw/@sap-cloud-sdk/cli)](https://www.npmjs.com/package/@sap-cloud-sdk/cli)
 
+## The SAP Cloud SDK CLI is going to be deprecated soon
+
+We're not doing any active development on the CLI. We'll replace it with a set of well maintained resources like a GitHub repository for the starter project, configuration guides, templates for pipeline and most popular SDK integrations. You can continue using the CLI as long as it serves your purpose but don't expect major updates, bugfixing, etc.
```

Comment:
```suggestion
We are not actively developing on the SAP Cloud SDK CLI anymore.
Instead, we will provide extensive setup guides for your projects, including examples for pipelines and the most popular SAP Cloud SDK integrations.
You can continue using the CLI as long as it serves your purpose but do not expect major updates, bug fixes, etc.
```

Reply:
I just don't exactly know if customers will have crocodile tears because of missing the CLI. But you're right, let's better roll out a replacement and then start talking about it.